### PR TITLE
Update gomplate to 3.11.3 and golang to 1.18.6

### PR DESCRIPTION
**Pull request by S-Sankara**
The version of `gomplate` is updated to v3.11.3.

### CODE REVIEW

File: dynamic/Dockerfile [hunk 1]
```diff
@@ -12,7 +12,7 @@ ENV LANG=C.UTF-8
 RUN export GO_VERSION="1.17.9" && \
 	export GO_SHAR256="9dacf782028fdfc79120576c872dee488b81257b1c48e9032d122cfdb379cca6 go${GO_VERSION}.linux-amd64.tar.gz" && \
 	export GO_BINARY="go${GO_VERSION}.linux-amd64.tar.gz" && \
-	export GOMPLATE_VERSION="3.10.0" && \
+	export GOMPLATE_VERSION="3.11.3" && \
 	export GOMPLATE_SHA256="f9a30d8e94b81eefbbe3455c21dc547ec0ebf0e010a809c72db617a4b37223a6  v${GOMPLATE_VERSION}.tar.gz" && \
```
Comment (tkohn) [inline dynamic/Dockerfile:16]: I guess there is also a new SHA available

File: static/Dockerfile [hunk 2]
```diff
@@ -12,7 +12,7 @@ ENV LANG=C.UTF-8
 RUN export GO_VERSION="1.17.9" && \
 	export GO_SHAR256="9dacf782028fdfc79120576c872dee488b81257b1c48e9032d122cfdb379cca6 go${GO_VERSION}.linux-amd64.tar.gz" && \
 	export GO_BINARY="go${GO_VERSION}.linux-amd64.tar.gz" && \
-	export GOMPLATE_VERSION="3.10.0" && \
+	export GOMPLATE_VERSION="3.11.3" && \
 	export GOMPLATE_SHA256="f9a30d8e94b81eefbbe3455c21dc547ec0ebf0e010a809c72db617a4b37223a6  v${GOMPLATE_VERSION}.tar.gz" && \
```
Comment (tkohn) [inline static/Dockerfile:16]: I guess there is also a new SHA available

**Review by tkohn — CHANGES_REQUESTED** [review]
Verify the sha of gomplate. You can also verify the images via `docker buildx build --progress=plain --no-cache --platform=linux/amd64 -t gcr.io/instana-agent-qa/tkohn-static-agent:1.1.635 .`

File: dynamic/Dockerfile (hunk 1 above)
Comment (S-Sankara) [reply dynamic/Dockerfile]: SHA updated

**Review by tkohn — APPROVED** [review]

**Pull request merged**